anchor-contract/Makefile

```diff
@@ -0,0 +1,10 @@
+build:
+	forge build
```

Comment:
I am curious. Can you install forge via npm?

Reply:
Might be possible through https://www.npmjs.com/package/@foundry-rs/easy-foundryup

Reply:
Sweet. Then the whole install/build/test process could be implemented similar to how we do in other packages. @Jovonni

Reply:
@Jovonni See https://www.twilio.com/blog/npm-scripts and https://docs.npmjs.com/cli/v7/configuring-npm/package-json#config and https://docs.npmjs.com/cli/v7/configuring-npm/npmrc for more information.